docs/sources/tempo/metrics-generator/cardinality.md

```diff
@@ -0,0 +1,83 @@
+---
+aliases:
+- /docs/tempo/latest/metrics-generator/cardinality
+title: Cardinality
+menuTitle: Cardinality
+description: What is cardinality and how it is impacted by metrics generation?
+weight: 100
+---
+
+# Cardinality
+
+Cardinality refers to the total combination of key/value pairs, such as labels and label values for a given metric series or log stream, and how many unique combinations they generate.
+For more information on cardinality, see the [What are cardinality spikes and why do they matter?](/blog/2022/02/15/what-are-cardinality-spikes-and-why-do-they-matter/) blog post.
+
+Because writes are series, high cardinality does not make a big difference. However, cardinality can have a major impact on querying where you iterate over a lot more things.
```

Comment:
```suggestion
Because writes to a TSDB database are in series, high cardinality does not make a big difference to performance at ingest. However, cardinality can have a major impact on querying where, the higher the cardinality, the more items are required to be iterated over.
```